src/odemis/util/conversion.py

```diff
@@ -165,6 +165,28 @@ def hex_to_frgba(hex_str, af=1.0):
     return rgba_to_frgba(hex_to_rgba(hex_str, int(af * 255)))
 
 
+def frgba_to_hex(frgba):
+    """ Convert a float RGBA value into a Hexadecimal colour representation.
+    If the alpha value is 1.0, it will output a 6-character hex (e.g., #RRGGBB).
+    Otherwise, it will output an 8-character hex (e.g., #RRGGBBAA).
+
+    :param frgba: (float, float, float, float) RGBA values in the range [0..1]
+    :return: Hexadecimal colour representation as a string
+    """
+    if len(frgba) != 4:
+        raise ValueError("Illegal FRGBA colour %s" % str(frgba))
```

Comment:
I would replace FRGBA with RGBA here, since it's technically just an rgba color, and give a bit more information:
```suggestion
        raise ValueError(f"RGBA {frgba} not accepted, must have length 4 but has length {len(frgba)}")
```